onnxruntime/core/providers/cpu/tensor/upsample.h

```diff
@@ -18,6 +18,10 @@ enum UpsampleMode {
 class UpsampleBase {
  protected:
   UpsampleBase(OpKernelInfo info) : scales_cached_(false) {
+    int start, end;
+    info.GetKernelDef().SinceVersion(&start, &end);
+    is_resize = (start == 10);
```

Comment:
>is_resize [](start = 4, length = 9)

should re-enable resize related test from onnx_backend_test_series.py?

Reply:
Good point, Done.